test/test_case.py

```diff
@@ -135,4 +134,4 @@ def test_match():
 
 
 def test_unicode():
-    assert unpackb(packb("foobar"), use_list=1) == b"foobar"
+    assert unpackb(packb(u"foobar"), use_list=1) == u"foobar"
```

Comment:
somehow makes more sense now. \o/

Reply:
are there enough tests that check all "old msgpack" behaviour as well as all "new msgpack" behaviour (str, bytes, unicode).

esp. that using it the old way still works with the new code, when the right options are given.

borgbackup depends on old format and as people have that in their backup repositories, we can't change it.